packages/models-library/src/models_library/api_schemas_directorv2/clusters.py

```diff
@@ -96,7 +97,7 @@ class ClusterGet(Cluster):
         alias="accessRights", default_factory=dict
     )
 
-    model_config = ConfigDict(populate_by_name=True)
+    model_config = ConfigDict(extra="allow", populate_by_name=True)
```

Comment:
why is this change needed? I can check it, maybe if you add my name or please copy it for me in the main issue

Reply:
If you remove it, a test fails because the `"forbid"` behavior is inherited from `BaseCluster` class.